Sources/ARKit-CoreLocation/Nodes/LocationNode.swift

```diff
@@ -182,4 +188,59 @@ open class LocationNode: SCNNode {
     func renderingOrder(fromDistance distance: CLLocationDistance) -> Int {
         return Int.max - 1000 - (Int(distance * 1000))
     }
+    
+    @available(iOS 11.0, *)
+    func stackNode(scenePosition: SCNVector3?, locationNodes: [LocationNode], stackingOffset: Float) {
```

Comment:
The function is called for each node that has the `shouldStackAnnotation` property set to `true` (there was a severe bug which made all annotations to be always stacked) when adding nodes to the scene. In `SceneLocationView`:
```swift
func addLocationNodeWithConfirmedLocation(locationNode: LocationNode) {
        ...
        if locationNode.shouldStackAnnotation {
            locationNode.stackNode(locationNodes: locationNodes, stackingOffset: stackingOffset)
        }
        ...
        sceneNode?.addChildNode(locationNode)
    }
```

I removed the first arg `scenePosition` because now I use SIMD assuming that the user position is centred at the origin and each node `simdWorldPosition`'s is a vector pointing at the node and we compute angles between nodes to check if the collides.

Each time we need to pass to the node a list with the already present nodes to check if the node is occluded.

The logic now is much simpler, fully using SIMD and with a docstring provided